src/lib/server/calendarUtils.ts

```diff
@@ -0,0 +1,95 @@
+import { CALENDLY_API_TOKEN, CALENDLY_API_URL, CALENDLY_USER_URI } from '$env/static/private';
+
+const AUTH_HEADER = `Bearer ${CALENDLY_API_TOKEN}`;
+
+export const getEventDetails = async (eventUuid: string) => {
+	const eventTypeResponse = await fetch(`${CALENDLY_API_URL}/event_types/${eventUuid}`, {
+		method: 'GET',
+		headers: {
+			Authorization: AUTH_HEADER,
+			'Content-Type': 'application/json'
+		}
+	});
+	if (!eventTypeResponse.ok) {
+		if (eventTypeResponse.status === 404) {
+			throw new Error('Event not found');
+		} else {
+			throw new Error(`Failed to fetch event details. Status: ${eventTypeResponse.status}`);
+		}
+	}
+	const eventTypeData: Calendly.EventTypeData = (await eventTypeResponse.json()).resource.map(mapCalendlyEventType);
+	return eventTypeData;
+};
+
+export const getAllEventTypes = async () => {
+	const eventTypesResponse = await fetch(`${CALENDLY_API_URL}/event_types?user=${CALENDLY_USER_URI}&active=true`, {
```

Comment:
added another uri param `active=true` to show only events that are actually active in Calendly -- they may want to keep around events that have already passed for attendance records, etc. so they can just leave them there and make them inactive.